src/components/migrationToV3.stories.mdx

```diff
@@ -0,0 +1,119 @@
+<Meta title="Migration to V3" />
+
+# Migration from 2.0 to 3.0
+
+Version 3 of the Amazon Chime React Component Library makes small changes to the interface of the library to make it more customizable.
+
+### IMPORTANT - Version 3 of the library also requires a dependency to version 3.0.0 or higher of the amazon-chime-sdk-js.
+
+In many cases you will need to adjust your application code because interface parameters to root providers like Meeting Provider are updated.
+
+## Installation
+
+Installation involves adjusting your package.json to depend on version 3.x.x
+
+```
+npm uninstall amazon-chime-sdk-component-library-react amazon-chime-sdk-js
+npm install --save amazon-chime-sdk-component-library-react amazon-chime-sdk-js
+```
+
+## What’s New
+
+- Updated props for `MeetingProvider`.
+- Updated `join` method props in `MeetingManager` component.
+
+### Meeting Provider prop change
+Meeting provider no longer takes `VideoDownlinkBandwidthPolicy`, `VideoUplinkBandwidthPolicy`, `simulcastEnabled`, `reconnectTimeoutMs`, and `keepLastFrameWhenPaused` as parameters.
+
+These properties can rather be updated by passing the `MeetingSessionConfiguration` via `meetingManager.join()` method. The following example uses `VideoDownlinkBandwidthPolicy` property as an example to showcase the change.
+
+If your code looks like this:
+```
+const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
```

Comment:
I have added the comment. I think writing this code would be overkill for just showing the diff.

Reply:
https://github.com/aws/amazon-chime-sdk-component-library-react/pull/751#discussion_r824052752

Reply:
Since, this is a react component library, please add an example in react format. I do not think it is an overkill for builders as well. In the current example, you have `useMeetingManager` which can be only used in a React functional component and not elsewhere.